### PR TITLE
feat(portal-next):  initialize storybook

### PR DESCRIPTION
**Pull request by jansobczynski**
## Issue

https://gravitee.atlassian.net/browse/APIM-4245

## Description

Add storybook

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-ubdcvblwic.chromatic.com)

### CODE REVIEW

File: gravitee-apim-portal-webui-next/.storybook/tsconfig.doc.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// This tsconfig is used by Compodoc to generate the documentation for the project.
+// If Compodoc is not used, this file can be deleted.
+{
+  "extends": "./tsconfig.json",
+  // Exclude all files that are not needed for documentation generation.
```
Comment (jourdiw) [inline gravitee-apim-portal-webui-next/.storybook/tsconfig.doc.json]: You can remove all the comments in this file

File: gravitee-apim-portal-webui-next/.storybook/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../tsconfig.app.json",
+  "compilerOptions": {
+    "types": ["node"],
+    "allowSyntheticDefaultImports": true,
+    "resolveJsonModule": true
+  },
+  "exclude": ["../src/test.ts", "../src/**/*.spec.ts"],
```
Comment (jourdiw) [inline gravitee-apim-portal-webui-next/.storybook/tsconfig.json]: There is no `test.ts` in the project, right?

File: gravitee-apim-portal-webui-next/package.json [hunk 3]
```diff
@@ -20,7 +20,9 @@
     "lint:license": "license-check-and-add check -f license-check-config.json",
     "lint:license:fix": "license-check-and-add add -f license-check-config.json -r $(date +%Y)",
     "lint": "yarn lint:license && yarn lint:eslint && yarn lint:styles && yarn lint:prettier",
-    "lint:fix": "yarn lint:license:fix && yarn lint:eslint:fix && yarn lint:styles:fix && yarn lint:prettier:fix"
+    "lint:fix": "yarn lint:license:fix && yarn lint:eslint:fix && yarn lint:styles:fix && yarn lint:prettier:fix",
+    "storybook": "ng run gravitee-apim-portal-webui-next:storybook",
+    "build-storybook": "ng run gravitee-apim-portal-webui-next:build-storybook"
```
Comment (jourdiw) [inline gravitee-apim-portal-webui-next/package.json]: what about naming it `storybook:build`?

File: gravitee-apim-portal-webui-next/package.json [hunk 4]
```diff
@@ -47,6 +49,13 @@
     "@angular-eslint/template-parser": "17.2.1",
     "@angular/cli": "17.2.3",
     "@angular/compiler-cli": "17.2.0",
+    "@chromatic-com/storybook": "^1.2.26",
```
Comment (jourdiw) [inline gravitee-apim-portal-webui-next/package.json]: Make sure to take out all `^`

File: gravitee-apim-portal-webui-next/.storybook/tsconfig.doc.json (hunk 1 above)
Comment (jansobczynski) [reply gravitee-apim-portal-webui-next/.storybook/tsconfig.doc.json]: Done

File: gravitee-apim-portal-webui-next/.storybook/tsconfig.json (hunk 2 above)
Comment (jansobczynski) [reply gravitee-apim-portal-webui-next/.storybook/tsconfig.json]: Done

File: gravitee-apim-portal-webui-next/package.json (hunk 3 above)
Comment (jansobczynski) [reply gravitee-apim-portal-webui-next/package.json]: Done

File: gravitee-apim-portal-webui-next/package.json (hunk 4 above)
Comment (jansobczynski) [reply gravitee-apim-portal-webui-next/package.json]: Done

**Review by jourdiw — APPROVED** [review]
Squash the commits and then it's good to merge 👍

**Pull request merged**